corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -0,0 +1,143 @@
+from datetime import datetime, timedelta
+
+from django.core.management.base import BaseCommand, CommandError
+
+from corehq.apps.es.client import ElasticMultiplexAdapter, get_client
+from corehq.apps.es.client import manager as es_manager
+from corehq.apps.es.exceptions import IndexNotMultiplexedException, TaskMissing
+from corehq.apps.es.registry import get_registry, registry_entry
+from corehq.apps.es.transient_util import doc_adapter_from_info
+from corehq.apps.es.utils import check_task_progress
+
+
+class ESSyncUtil:
+
+    """
+    This class contains methods to support the reindex process of multiplexed indices in HQ
+    """
+
+    def __init__(self):
+        self.es = get_client()
+
+    def start_reindex(self, cname):
+
+        self.adapter = self.get_adapter(cname)
```

Comment:
I don't think this should be assigned to the instance. It would make the usage clearer it was kept as a local var and passed to methods that need it (which I think is only `perform_cleanup`.

Reply:
https://github.com/dimagi/commcare-hq/pull/32370/commits/f8c0b0fc0cd3353263e76a40e2ad6cbaee0092f1